src/calculateBullsAndCows.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const calculateBullsAndCows = (userAnswer, generatedNumber) => {
+  const result = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  for (let i = 0; i < userAnswer.length; i++) {
+    if (userAnswer[i] === generatedNumber[i]) {
+      result.bulls += 1;
+      continue;
+    }
+
+    if (generatedNumber.includes(userAnswer[i])) {
+      result.cows += 1;
+    }
+  }
```

Comment:
Try to avoid using old traditional for loops. Instead, use specific ones like filter/map/foreach/reduce. Here fits the best reduce.